index.html

```diff
@@ -56,105 +56,37 @@ <h1>I'm Soban.<br> Glad to see you!</h1>
 
         <section id="workspace">
 
-            <div class="works">
-                <img src="./images/Snapshoot Portfolio.svg" alt="Tonic" class="workimage">
-                <img src="./images/Snapshoot Portfolio 1.svg" alt="Nature chart" class="deskimg">
-                <div class="worktext">
-                    <h2>Tonic</h2>
-                    <div class="desc">
-                        <p class="descA">CANOPY</p>
-                        <img src="./images/Counter.svg" alt="counter">
-                        <p class="grey">Back End Dev</p>
-                        <img src="./images/Counter.svg" alt="counter">
-                        <p class="grey">2015</p>
+            <div class="pop-work">
+                <div class="pop-card">
+                    <div class="pop-head">
+                        <h2 id="poptitle"></h2>
+                        <img src="./images/X-popup.svg" alt="close button" id="closebtn">
```

Comment:
- [ ] It is required according to the project requirements that aready created HTML code for the projects list should be removed, and instead it should be added via JavaScript. Kindly remove all the previous code and add them via JavaScript.

### Project Requirements
![project-rqt](https://github.com/SnakyMz/Portfolio/assets/26597432/b8bf4323-b4f0-405f-b034-e1e2cc990d12)

Reply:
<div class="pop-work"> is not old projects file. It is the popup window layout. And its elements and values are dynamically assigned using javascript. Please have a closer look at it again